tests/HTLC.test.ts

```
@@ -24,13 +24,13 @@ import {
   BitcoinWallet,
 } from "@catalogfi/wallets";
 import { regTestUtils } from "./bitcoin/regtest";
-import axios from "axios";
 
-const nodeUrl = "http://127.0.0.1:8547/rpc";
+export const STARKNET_DEVNET_URL = "http://127.0.0.1:8547/rpc";
+export const BITCOIN_NODE_URL = "http://127.0.0.1:18443"
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)